tests/test_vega.py

```diff
@@ -6,7 +6,17 @@
 from dvc_render.vega import BadTemplateError, VegaRenderer
 from dvc_render.vega_templates import NoFieldInDataError, Template
 
-# pylint: disable=missing-function-docstring
+# pylint: disable=missing-function-docstring, C1803
+
+
+def test_init_empty():
+    renderer = VegaRenderer(None, None)
+
+    assert renderer.datapoints == []
+    assert renderer.name == ""
+    assert renderer.properties == {}
+
+    renderer.partial_html()
```

Comment:
Should we check the contents of the resulting template?

Reply:
done